src/figures/figureW.jl

```diff
@@ -77,6 +91,10 @@ function plotDepletionSynergy(IgGXidx::Int64, IgGYidx::Int64, weights::Vector; L
     if c1q
         X = vcat(X, Kav_df[!, :C1q]' * IgGC)
     end
+    if Neutralization
+        deleteat!(weights, 6)
+        Yname = murine ? murineIgG[IgGYidx-1] : humanIgG[IgGYidx-1]
```

Comment:
Why `IgGYidx-1` ?

Reply:
For the HIV data, the second "condition" is IgG1D265A. Because that IgG is not included in murine affinities, it does not have an indexing option. Instead, I had IgG1 and IgG2a compared which would normally be index 1 and 2, but when using HIV it has to be 1 and 3. The minus 1 was to get the naming to work alright. The naming wouldn't be correct if we were trying to compare IgG1 and IgG1D265A in the synergy plot. This is a sloppy fix, so I could add IgG1D265A to murine affinities, but I didn't want to break basically everything that relies on those indices. @cyrillustan

Reply:
When you use `IgGXidx=1` and `IgGYidx=2`, you will get IgG1 and IgG2a for mice, and IgG1D265A is never an option in `plotDepletionSynergy`.  Why the indices have to be 1 and 3? Also, just to confirm, you don't take neutralization into account when calculating synergy.

Reply:
Ah, I see. I'll double check that it doesn't work for 1 and 3 then and get rid of the -1

Reply:
Ok, so when I took out the -1 and ran plotDepletionSynergy with indices 1 and 3, it still worked. This is odd to me because we don't have HIV data with IgG2b so why would this still work?

Reply:
`plotDepletionSynergy()` never uses the data frame we regressed on. It calculates the numbers fresh, with only the weights from our model. After all, the plot is all about the simulated results